src/Faker/Provider/ne_NP/Payment.php

```diff
@@ -0,0 +1,111 @@
+<?php
+
+namespace Faker\Provider\ne_NP;
+
+class Payment extends \Faker\Provider\Payment
+{
+    /**
+     * List of commercial banks sorted in alphabetical order.
+     *
+     * @var string[]
+     */
+    protected static $commercialBanks = ['Agricultural Development Bank', 'Bank Of Kathmandu', 'Century Commercial Bank', 'Citizens Bank International', 'Civil Bank', 'Everest Bank', 'Global IME Bank', 'Himalayan Bank', 'Kumari Bank', 'Laxmi Bank', 'Machhapuchchhre Bank', 'Mega Bank Nepal', 'Nabil Bank', 'Nepal Bangladesh Bank', 'Nepal Bank', 'Nepal Credit & Commerce Bank', 'Nepal Investment Bank', 'Nepal SBI Bank', 'NIC ASIA Bank', 'NMB Bank', 'Prabhu Bank', 'Prime Commercial Bank', 'Rastriya Banijya Bank', 'Sanima Bank', 'Siddhartha Bank', 'Standard Chartered Bank Nepal', 'Sunrise Bank'];
```

Comment:
Can we put this multiline just like other providers?

Reply:
And please provice a source for the data! :)